src/c_shim.rs

```diff
@@ -113,4 +113,4 @@ pub unsafe extern "C" fn memset(s: *mut c_void, c: i32, n: usize) -> *mut c_void
 #[no_mangle]
 pub unsafe extern "C" fn iswspace(c: c_int) -> bool {
     char::from_u32(c as u32).map_or(false, |c| c.is_whitespace())
-}
+}
```

Comment:
Can you add this newline at the end of the file back? There is an editor setting in VSCode to add newlines at end of files on save.